README.md

```diff
@@ -6,23 +6,24 @@ ___________________________________________
 
 ## How install Lutris Beta
 1. Install Flathub Beta
-   1. Open the application called 'Terminal'
-   2. In this application run the command:
+
+Enter this command in a terminal window
```

Comment:
IMO this is redundant, too. Monospace text here is obviously a command to run.

Reply:
## How install Lutris Beta
1. Install Flathub Beta

**Command:**
```
flatpak remote-add --user flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak update --appstream
```